### PR TITLE
Atlanta update

### PR DESCRIPTION
**Pull request by s-bessey**
updates to run the atlanta model

### CODE REVIEW

File: titan/settings/atlanta/partnership.yml [hunk 1]
```diff
@@ -41,4 +41,7 @@ partnership:
       MSM:
         insertive: 0.0011
         receptive: 0.0138
-        versatile: 0.00745
+        versatile: 0.
```
Comment (eldu) [inline titan/settings/atlanta/partnership.yml:44]: Should this be non-zero?

File: titan/model.py [hunk 2]
```diff
@@ -387,6 +387,7 @@ def exit(self):
                     # agent ages out of model
                     if agent.age > exit.age:
                         self.exits[strategy.exit_class].append(agent)
+                        break
```
Comment (eldu) [inline titan/model.py:390]: Sorry this is on a somewhat relevant row, but not where I wanted it to be? Github won't let me choose a more obvious row.

Anyway! This whole `if case` feels very much a like switch statement structure! 

https://docs.python.org/3.10/whatsnew/3.10.html#pep-634-structural-pattern-matching

**Review by eldu — APPROVED** [review]
I added in some comments! I think it looks great! 

Though, I'm not as familiar with TITAN and some of the number changes from previously don't make as much sense to me. I guess it's more so for alignment of the data?

File: titan/model.py (hunk 2 above)
Comment (s-bessey) [reply titan/model.py:390]: We thought about switch statements, but we want to stay compatible with 3.9, which doesn't allow switch statements 😞

**Pull request merged**